tasks.py

```diff
@@ -38,54 +38,52 @@ def clean(bower=False, node=False):
     ]
     for pattern in patterns:
         print('Removing {0}'.format(pattern))
-        run('cd {0} && rm -rf {1}'.format(ROOT, pattern))
+        lrun('rm -rf {0}'.format(ROOT, pattern))
```

Comment:
Isn't it dangerous to launch a `rm -rf` at an unidentified place? + `format()` contains now an extra parameter.